apps/data/priv/repo/seeds.exs

```diff
@@ -9,3 +9,50 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+project_params_1 = %{
+  name: "Elixir School",
+  tags: ["elixir"],
+  url: "https://github.com/elixirschool/elixirschool",
+}
+
+project_params_2 = %{
+  name: "Extracurricular",
+  tags: ["elixir", "open source"],
+  url: "https://github.com/elixirschool/extracurricular",
+}
+
+project_params_3 = %{
+  name: "Appendix",
+  tags: ["elixir", "open source", "blog"],
+  url: "https://github.com/elixirschool/appendix",
+}
+
+{:ok, elixir_school}   = Data.Projects.insert(project_params_1)
+{:ok, extracirricular} = Data.Projects.insert(project_params_2)
+{:ok, appendix}        = Data.Projects.insert(project_params_3)
+
+opportunity_params_1 = %{
+  title: "Blog post on Git Bisect",
+  level: 1,
+  url: "https://github.com/elixirschool/appendix/issues/1",
+  project_id: appendix.id
+}
+
+opportunity_params_2 = %{
+  title: "Translate to pig latin",
+  level: 1,
+  url: "https://github.com/elixirschool/elixirschool/issues/1",
+  project_id: elixir_school.id
+}
+
+opportunity_params_3 = %{
+  title: "Rewrite it in Node/Ember lol",
```

Comment:
Let's stick to realistic seed data:  "Implement API for sorting" or something like that